frontend/app/model/Destination.scala

```diff
@@ -0,0 +1,16 @@
+package model
+
+import com.netaporter.uri.Uri
+import model.RichEvent.RichEvent
+
+sealed trait Destination
+
+case class EventDestination(event: RichEvent, iframeUrl: Uri) extends Destination {
+  def iframeHeight = {
```

Comment:
This can be a `val` (not a `def`), because it never changes in the scope of this class!